### PR TITLE
 opengrm-ngram: init at 1.3.11, phonetisaurus: init at 2020-07-31

### PR DESCRIPTION
**Pull request by Mic92**
###### Motivation for this change

This is a preparation for [rhasspy](https://github.com/rhasspy/rhasspy).

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/phonetisaurus/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{ stdenv
+, fetchFromGitHub
+, openfst
+, pkg-config
+, python3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "phonetisaurus";
+  version = "2020-07-31";
```
Comment (Mic92) [inline pkgs/development/libraries/phonetisaurus/default.nix:10]: Needs to be unstable to be compatible with openfst.

**Comment by Mic92** [comment]
This is a preparation for [rhasspy](https://github.com/rhasspy/rhasspy).

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -27138,6 +27138,10 @@ in
 
   openfst = callPackage ../development/libraries/openfst {};
 
+  opengrm-ngram = callPackage ../development/libraries/opengrm-ngram {};
+
+  phonetisaurus = callPackage ../development/libraries/phonetisaurus {};
```
Comment (Mic92) [inline pkgs/top-level/all-packages.nix:27143]: All three are related so I keep them together.

File: pkgs/development/libraries/phonetisaurus/default.nix (hunk 1 above)
Comment (Mic92) [reply pkgs/development/libraries/phonetisaurus/default.nix:10]: I asked for a release here: https://github.com/AdolfVonKleist/Phonetisaurus/issues/61

File: pkgs/development/libraries/opengrm-ngram/default.nix [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  nativeBuildInputs = [ autoreconfHook ];
+
+  buildInputs = [ openfst ];
+
+  enableParallelBuilding = true;
+
+  meta = with stdenv.lib; {
+    description = "Library to make and modify n-gram language models encoded as weighted finite-state transducers";
+    homepage = "http://www.openfst.org/twiki/bin/view/GRM/NGramLibrary";
+    license = licenses.mit;
```
Comment (mweinelt) [inline pkgs/development/libraries/opengrm-ngram/default.nix:21]: > It is an open source project distributed under the Apache license. 

```suggestion
    license = licenses.asl2;
```

As seen on the homepage.

File: pkgs/development/libraries/phonetisaurus/default.nix [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ python3 openfst ];
+
+  meta = with stdenv.lib; {
+    description = "Framework for Grapheme-to-phoneme models for speech recognition using the OpenFst framework";
+    inherit (src.meta) homepage;
```
Comment (mweinelt) [inline pkgs/development/libraries/phonetisaurus/default.nix:27]: Where does that point to?

File: pkgs/development/libraries/phonetisaurus/default.nix [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    license = licenses.mit;
```
Comment (mweinelt) [inline pkgs/development/libraries/phonetisaurus/default.nix:28]:
```suggestion
    license = licenses.bsd3;
```

**Comment by mweinelt** [comment]
Result of `nixpkgs-review pr 94465` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
<br>- opengrm-ngram
<br>- phonetisaurus
</details>

**Review by mweinelt — CHANGES_REQUESTED** [review]
Some minor nits.

File: pkgs/development/libraries/phonetisaurus/default.nix (hunk 4 above)
Comment (Mic92) [reply pkgs/development/libraries/phonetisaurus/default.nix:27]: To the github repo.

Comment (mweinelt) [reply pkgs/development/libraries/phonetisaurus/default.nix:27]: I don't understand how. Is this something the `fetchFromGitHub` function provides?

File: pkgs/development/libraries/opengrm-ngram/default.nix [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  nativeBuildInputs = [ autoreconfHook ];
+
+  buildInputs = [ openfst ];
+
+  enableParallelBuilding = true;
+
+  meta = with stdenv.lib; {
+    description = "Library to make and modify n-gram language models encoded as weighted finite-state transducers";
+    homepage = "http://www.openfst.org/twiki/bin/view/GRM/NGramLibrary";
+    license = licenses.asl2;
```
Comment (mweinelt) [inline pkgs/development/libraries/opengrm-ngram/default.nix:21]:
```suggestion
    license = licenses.asl20;
```

File: pkgs/development/libraries/phonetisaurus/default.nix (hunk 4 above)
Comment (Mic92) [reply pkgs/development/libraries/phonetisaurus/default.nix:27]: yes.

**Comment by Mic92** [comment]
rebased!

**Comment by mweinelt** [comment]
merged!

**Pull request merged**